docs/decisions/0047-azure-open-ai-connectors.md

```diff
@@ -0,0 +1,188 @@
+
+### Phases of the transition
+
+- **Phase 1**: Add new OpenAI SDK APIs to the current OpenAI connector and keep the Azure OpenAI APIs using the last Azure SDK.
+- **Phase 2**:
+  - Create a new connector for Azure OpenAI services using the new Azure SDK
+  - Deprecate all Azure OpenAI APIs in the `OpenAI` connector pointing to new `AzureOpenAI` connector
+  - Remove Azure SDK dependency from the OpenAI connector.
+  - Add `AzureOpenAI` connector to the `Microsoft.SemanticKernel` meta package.
+- **Phase 3**: Deprecate all legacy `OpenAI APIs` in the `OpenAI` connector pointing to new `Options` APIs.
+- **Phase 4**: Remove all legacy APIs from the OpenAI connector.
+
+### Impact
+
+Pros:
+
+- Minimal breaking changes for developers using the current OpenAI connector.
+- Clear separation of concerns between OpenAI and Azure OpenAI connectors.
+
+Cons:
+
+- Since `SemanticKernel.Connectors.AzureOpenAI` and `SemanticKernel.Connectors.OpenAI` share a same dependency of different versions, both packages cannot be used in the same project and a strategy will be needed when deploying both connectors:
+- Added dependency for both `Azure OpenAI 1.0-beta17` and `OpenAI 2.0-beta1`.
+
+### Dependency Management Strategies
+
+1. Use only one of the connectors in the same project, some modifications will be needed to accommodate `Concepts` and other projects that shares OpenAI and AzureOpenAI examples.
+2. Hold AzureOpenAI connector implementation until we are ready to break (exclude) all Azure APIs in OpenAI connector.
+3. Deploy a new project with a new namespace for `Azure.AI.OpenAI.Legacy 1.0.0-beta.17` and update our `SemanticKernel.Connectors.OpenAI` to use this new namespace to avoid version clashing on the `Azure.AI.OpenAI` namespace.
+
+## Option 2 - Independent Connectors from Start.
+
+This option is focused on creating fully independent connectors for OpenAI and Azure OpenAI services since the start with all breaking changes needed to achieve that.
+
+Impact:
+
+- `Azure` extension methods will be removed from `SemanticKernel.Connectors.OpenAI` namespace to avoid clashing with same names introduced in the new `SemanticKernel.Connectors.AzureOpenAI`.
+
+- All azure specific services like `AzureOpenAIChatCompletion`, `AzureOpenAITextGeneration` services will be obsoleted and throw not supported exceptions when used from `SemanticKernel.Connectors.OpenAI` connector.
+
+### Impact
+
+Pros:
+
+- Clear separation of concerns between OpenAI and Azure OpenAI connectors.
+- Small breaking changes for developers focused on OpenAI specific APIs.
+- Faster transition to the new OpenAI SDK and Azure OpenAI SDK.
+
+Cons:
+
+- Large breaking changes for developers using the current OpenAI connector for Azure.
+- ⚠️ May have the same problem with the dependency management as in Option 1 when using both connectors in the same project may conflict due to the shared dependency of different OpenAI SDK versions.
```

Comment:
Can you expand on this so it is clear to all readers what the issue is.

Also ad a statement explaining what our position will be.